kyo-bench/src/main/scala/kyo/bench/TRefMultiBench.scala

```diff
@@ -13,7 +13,7 @@ class TRefMultiBench(parallelism: Int) extends Bench.ForkOnly(parallelism):
 
         STM.runtime[IO].flatMap { stm =>
             for
-                refs   <- stm.commit(Seq.fill(parallelism)(stm.TVar.of(0)).sequence)
+                refs   <- Seq.fill(parallelism)(stm.commit(stm.TVar.of(0))).sequence
```

Comment:
I noticed the cats-effect implementation was different from kyo and zio, not committing each new var separately